app/controllers/comments_controller.rb

```diff
@@ -6,12 +6,17 @@ def new
     @post_id = 1 # This part should be revised
   end
 
+  # TODO: refactor this code
   def create
+    author_id = params[:author_id]
+    post_id = params[:post_id]
+    text = params[:text]
+
     @comment = Comment.new(
```

Comment:
* Kindly Create a comment on behalf of the `current_user` (use the method that you created in your `ApplicationController`)as stated in the [ project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Create%20a%20comment%20on%20behalf%20of%20the%20current_user%20(use%20the%20method%20that%20you%20created%20in%20your%20ApplicationController).). HINTS 💭 `@comment = @current_user.comments.new(params)`